### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

File: components/Card.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  line-height: 16px;
+  font-weight: 700;
+  margin-bottom: 0.5rem;
+}
+.card-ecom .card-desc {
+  font-size: 16px;
+  line-height: 16px;
+  font-weight: 400;
+  margin-bottom: 0.5rem;
+  color: var(--primary-light-color);
```
Comment (ApurvaSawant11) [inline components/Card.css:69]: `.card-desc` text is a bit difficult to read. Try to use darker color or a lighter background.

**Comment by aayushjain-code** [comment]
good work keep going!

File: components/Card.css (hunk 1 above)
Comment (MuskanChhatrasal) [reply components/Card.css:69]:
```suggestion
  color: var(--primary-light-color);
```
Thankyou. Will do that

**Review by animeshgarai09 — COMMENTED** [review]
Try to use `rem` values

**Comment by MuskanChhatrasal** [comment]
Thankyou

**Comment by MuskanChhatrasal** [comment]
Okayy will see that. Thankyou

**Comment by MuskanChhatrasal** [comment]
Thankyou

**Pull request merged**